### ISSUE TITLE
Request: Activeloop

### ISSUE THREAD

**Issue by ancilcleetus** [simple-icons/simple-icons#11387]
### Brand Name

Activeloop

### Website

https://www.activeloop.ai/

### Popularity Metric

Activeloop provides Deep Lake - an Enterprise-Grade Database for AI. Deep Lake is a multi-modal database for AI. It can store all types of data (vectors, images, texts, videos etc.). It can store, query, version & visualize any AI data. It is used to build data-driven applications with LLMs (Large Language Models), LangChain etc. It can stream data in real-time to PyTorch or TensorFlow.

It is ranked 546,640 in SimilarWeb. Its [GitHub repo](https://github.com/activeloopai/deeplake) has 7.9k stars. The star history is on a consistently upward trajectory.

### Official Resources for Icon and Color

svg: https://www.activeloop.ai/static/logo-9fa16b423d87d3f5f877149da3ab5c9e.svg

**Comment by adamrusted** [comment]
Under the 'intellectual property rights' section - they explicitly state the license they grant is only for you to view their website, meaning we can't use anything sourced from there without their explicit permission. Would you like to reach out to them @ancilcleetus? They may even be able to provide us with a decent monochrome variation, should the be willing for us to include it.

**Comment by ancilcleetus** [comment]
@adamrusted Sent a mail to Activeloop seeking permission to use the icon. The [link to this issue](https://github.com/simple-icons/simple-icons/issues/11387) is also cited in the mail.

**Comment by ancilcleetus** [comment]
Content of the mail:

Hi Activeloop Support Team,

I'm using your Deep Lake database for doing some personal LLM projects.

I want to add Activeloop's icon in my GitHub Profile Readme (this Readme lists all the libraries, tools & frameworks that I'm using).

Since the terms given in your website explicitly state that the license you grant is only for viewing the website, I want your explicit permission to add the icon in my GitHub repo.

I have also created a request in Simple Icons GitHub repo to create an icon for Activeloop so that many people can use your icon listed in [simpleicons.org](http://simpleicons.org/) website.

Relevant Links are given below:

My GitHub repo: https://github.com/ancilcleetus

Simple Icons Website: https://simpleicons.org/

Icon request in Simpleicons repo: https://github.com/simple-icons/simple-icons/issues/11387

If you give permission to use your icon, it will be really helpful.


With Best Regards,

Ancil Cleetus
Artificial Intelligence Engineer
Kochi, Kerala, India

**Comment by ancilcleetus** [comment]
Content of the mail:

Hi Activeloop Support Team,

I'm using your Deep Lake database for doing some personal LLM projects.

I want to add Activeloop's icon in my GitHub Profile Readme (this Readme lists all the libraries, tools & frameworks that I'm using).

Since the terms given in your website explicitly state that the license you grant is only for viewing the website, I want your explicit permission to add the icon in my GitHub repo.

I have also created a request in Simple Icons GitHub repo to create an icon for Activeloop so that many people can use your icon listed in [simpleicons.org](http://simpleicons.org/) website.

Relevant Links are given below:

My GitHub repo: https://github.com/ancilcleetus

Simple Icons Website: https://simpleicons.org/

Icon request in Simpleicons repo: https://github.com/simple-icons/simple-icons/issues/11387

If you give permission to use your icon, it will be really helpful.


With Best Regards,

Ancil Cleetus
Artificial Intelligence Engineer
Kochi, Kerala, India

**Comment by mikayelh** [comment]
hi @adamrusted - @ancilcleetus, thank you so much for the initiative. I'm from team Activeloop, hereby granting explicit permission for it to be included! Uploading both color and monochrome versions - our preference would be to include the color version as well due to the logo symbol being overall very popular, so it may be confusing for people. 

Also attaching the mascot icon - it's pretty commonly used interchangeably for the package. Let me know if anything else is needed on our end.

![activeloop black symbol](https://github.com/simple-icons/simple-icons/assets/47828027/beab2c32-ce2e-4f2b-becc-bce4e873b1f1)
![activeloop white symbol](https://github.com/simple-icons/simple-icons/assets/47828027/55790769-4138-4495-b167-475b0f4d943d)
![activeloop orange symbol](https://github.com/simple-icons/simple-icons/assets/47828027/0288adc1-0e6b-4c1e-9239-6a7c19d99c1a)

![tensie activeloop mascot for deeplake package](https://github.com/simple-icons/simple-icons/assets/47828027/732d0d39-f877-46a1-934a-e333aca95b61)

Warmly,
mikayelh
